### ISSUE TITLE
Add some wording to the reporting screens to let users know when they are reporting remote users.

### ISSUE THREAD

**Issue by KuJoe** [mastodon/mastodon#23148]
### Pitch

A lot of users are unaware of how Mastodon and other decentralized platforms work and with every server/instance having their own rules it would be good if when a user attempts to report a remote user and selects their servers rules as the reason they are given a message/warning informing them that the rules may not apply to the user they are reporting since each server has their own moderation staff and rules.

Some users want their admin to police other servers which is not something we should be doing or they should be expecting.

### Motivation

Users who are unaware that every Mastodon server/instance has their own moderation staff and rules will report posts/users that only violate their local rules, but not the rules the posts/users are on. This is normally not done maliciously, so educating them with a bit of text in the report screens that rules are specific to their server/instance and they have selected a post/user on a remote server could go a long way in teaching them how to use filters, mutes, and blocks.

**Comment by trwnh** [comment]
i think the real issue with reporting remote users is not that the user is remote, it's that the report gets forwarded remotely, and specifically that this is up to the user instead of up to the admin/mod team. admins *should* be expected to moderate remote users on their local server based on local rules. i don't think it's wrong for users to expect local moderation. what's wrong is when users expect remote moderation according to local rules.

**Comment by KuJoe** [comment]
> i think the real issue with reporting remote users is not that the user is remote, it's that the report gets forwarded remotely, and specifically that this is up to the user instead of up to the admin/mod team (related: #18518 ). admins *should* be expected to moderate remote users on their local server based on local rules. i don't think it's wrong for users to expect local moderation. what's wrong is when users expect remote moderation according to local rules. local users shouldn't have to know the remote rules of every server they interact with.

I agree, but setting expectations for the user to let them know that certain rules are specific to the Local timeline only and not for the Federated timeline can save a lot of time for both the users and the moderation team.

A good example is if you have a region specific server you can't be expected that the Federated timeline will abide by rules specific to your region.

Maybe a better approach is to have certain rules be marked as "local only" so they do not appear as reporting options for remote users?

**Comment by trwnh** [comment]
> Maybe a better approach is to have certain rules be marked as "local only" so they do not appear as reporting options for remote users?

I'm not sure that would be better, or that it would work at all -- you need to be able to report remote users to your local admin for violating local rules so that a local action can be taken. In any case, the attached rules don't federate; only the comment does. Put another way: *all* rules are "local only".

**Comment by KuJoe** [comment]
I didn't know that, appreciate the clarification there. In that case I can make some adjustments to my rules to let users know they don't apply to other servers so hopefully we get less reports for them.
